server/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -74,6 +76,11 @@
 
 public class MapperService extends AbstractIndexComponent implements Closeable {
 
+    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(
+        LogManager.getLogger(QueryShardContext.class));
+    public static final String TYPES_DEPRECATION_MESSAGE = "[types removal] Using the _type field " +
+        "in queries and aggregations is deprecated, prefer to use a field instead.";
```

Comment:
nit: If this is only applicable for v7 compatibility mode, should we cal this constant: `TYPES_DEPRECATION_MESSAGE_V7`?

Reply:
Should we remove `_type` field from [SORTED_META_FIELDS](https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/index/mapper/MapperService.java#L113) constant?